services/crypto_create.proto

```diff
@@ -175,4 +173,9 @@ message CryptoCreateTransactionBody {
      * cover auto-renewal fees.
      */
     AccountID auto_renew_account = 19;
+
+    /**
+     * EOA 20-byte address to create.
```

Comment:
note keccak-256 hash of a ECDSA_SECP256K1 primitive key

Reply:
Added.